pkg/controller/etcdproxy/etcdproxy_controller.go

```diff
@@ -287,6 +287,15 @@ func (c *EtcdProxyController) syncHandler(key string) error {
 		return nil
 	}
 
+	// Deploy Server Etcd Proxy certificates.
```

Comment:
General question. Why are we creating individual configmaps and secrets for each etcdproxy we create instead of referencing them by name?